tests/acceptance/server.js

```diff
@@ -0,0 +1,23 @@
+const HttpServer = require('../test-utils/server');
+const path = require('path');
+
+/**
+ * Initalizes an http server for the provided fixture context
+ * @param {*} ctx the context from the fixture
+ */
+export async function setupServer (ctx) {
```

Comment:
since we're starting fresh, could we try using the testcafe server infrastructure? I ran into issues trying to parallelize our acceptance tests in the past because we were hardcoding it to a single port
https://testcafe.io/documentation/402641/reference/testcafe-api/runner